shared/facebook/facebook.properties.po

```diff
@@ -1,65 +1,64 @@
-#. extracted from ./shared/facebook/facebook.properties
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2014-07-28 20:49+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2015-06-18 20:37+0200\n"
 "Language: nd\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Translate Toolkit 1.12.0-alpha1\n"
+"X-Generator: Poedit 1.8.1\n"
 "X-Accelerator-Marker: &\n"
 "X-Merge-On: location\n"
+"Last-Translator: \n"
+"Language-Team: \n"
 
 #: close
 msgid "Close"
-msgstr ""
+msgstr "vala"
```

Comment:
check the case, it should be "Vala".

Oh i can also mention you like this @Mabheks . That way you get notified of the special @ mention